packages/styling/src/interfaces/ITheme.ts

```diff
@@ -5,5 +5,10 @@ import { ISemanticColors } from './ISemanticColors';
 export interface ITheme {
   palette: IPalette;
   fonts: IFontStyles;
-  semanticColors?: ISemanticColors;
+  semanticColors: ISemanticColors;
 }
+export interface IPartialTheme {
+  palette: Partial<IPalette>;
```

Comment:
You're right. I had done this before I pulled change that added a default palette.